### DISCUSSION TITLE
dify reported an error when using openai recall test to be a bytes-like object or ASCll string, not 'list'Help

### DISCUSSION THREAD

**Discussion by shanaozi** [langgenius/dify#6752] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

dify reported an error when using openai recall test to be a bytes-like object or ASCll string, not 'list'

**Comment by crazywoola** [comment]
What's your current version?

**Reply by shanaozi** [reply]
Pulled the latest code of maste
Which version do you mean?

**Reply by shanaozi** [reply]
0.6.15
![Uploading image.png…]()
